core/src/mindustry/world/blocks/distribution/ItemBridge.java

```diff
@@ -167,6 +167,37 @@ public void drawConfigure(){
             Draw.reset();
         }
 
+        @Override
+        public void drawSelect(){
+            for(int i = 1; i <= range; i++){
+                for(int j = 0; j < 4; j++){
+                    Tile other = tile.getNearby(Geometry.d4[j].x * i, Geometry.d4[j].y * i);
+                    if(!linkValid(tile,other,false)) continue;
+                    boolean linked = other.pos() == link;
+                    if(!linked && !(other.<ItemBridgeEntity>ent().link == tile.pos())) continue;
+                    float tx = tile.drawx(), ty = tile.drawy(), ox = other.drawx(), oy = other.drawy();
+                    float alpha = Math.abs((linked ? 100 : 0)-(Time.time() * 2) % 100f) / 100;
+                    float x = Interpolation.linear.apply(ox, tx, alpha);
+                    float y = Interpolation.linear.apply(oy, ty, alpha);
+
+                    Tile link = linked ? tile : other;
+                    Tile otherLink = linked ? other : tile;
+
+                    Draw.color(Color.valueOf("444444"));
+                    Lines.stroke(2.5f);
+                    Lines.circle(ox, oy, 2f);
```

Comment:
Circles don't really fit here, and may reduce performance. Draw a 45° rotated square instead (`Lines.poly()`)